ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -261,6 +270,7 @@ const triggerSet: TriggerSet<Data> = {
         insideOut: {
           en: 'Max Melee => Under',
           cn: '钢铁 => 月环',
+          ko: '밖으로 => 안으로',
```

Comment:
`Max Melee`는 칼끝딜로 번역하면 더 좋습니다.
![image](https://github.com/user-attachments/assets/204809ac-944e-46fe-90c3-cacecd67245b)

```suggestion
          ko: '칼끝딜 => 안으로',
```

Reply:
루비칸테같은경우는 칼끝딜만 하면 끝나는 구조라 칼끝딜 번역이 맞으나

이 기믹은 전형적인 루나-다이나모형태라 안-밖/밖-안 으로 움직여야하는 형태입니다.
캐스터힐러직군에게는 어감이 잘 와닿지 않다고 생각했습니다.